### ISSUE TITLE
Results from paper

### ISSUE THREAD

**Issue by ehedlin** [microsoft/MeshTransformer#10]
I am unable to get the results on the human3.6m dataset as published in the paper. Im getting 55.1584 MPJPE and 39.2701 PA-MPJPE on models/metro_release/metro_h36m_state_dict.bin

I am loading the model in the same way as in metro/tools/end2end_inference_bodymesh.py and metro/tools/run_metro_bodymesh.py. Is that not the optimal way? 

Other than that I am taking tight crops around the ground truth 2d keypoints (with a 1.2x margin) and normalizing.

**Comment by kevinlin311tw** [comment]
Thanks for raising the issue. 

I re-run the evaluation with the model checkpoint. But I couldn't obtain the numbers you mentioned above. Could you please share your commands here? It would be helpful to identify the issue.

I use the following command:
```
python -m torch.distributed.launch --nproc_per_node=8 metro/tools/run_metro_bodymesh.py --val_yaml human3.6m/valid.protocol2.yaml --arch hrnet-w64 --per_gpu_eval_batch_size 30 --run_eval_only --num_worker 24 --resume_checkpoint models/metro_release/metro_h36m_state_dict.bin --num_hidden_layers 4 --num_attention_heads 4 --input_feat_dim 2051,512,128 --hidden_feat_dim 1024,256,128
```

The corresponding output log:
```
2021-06-08 20:53:43,911 METRO INFO: Using 8 GPUs
2021-06-08 20:53:48,330 METRO INFO: Update config parameter num_hidden_layers: 12 -> 4
2021-06-08 20:53:48,383 METRO INFO: Update config parameter hidden_size: 768 -> 1024
2021-06-08 20:53:48,440 METRO INFO: Update config parameter num_attention_heads: 12 -> 4
2021-06-08 20:53:49,899 METRO INFO: Init model from scratch.
2021-06-08 20:53:49,956 METRO INFO: Update config parameter num_hidden_layers: 12 -> 4
2021-06-08 20:53:50,008 METRO INFO: Update config parameter hidden_size: 768 -> 256
2021-06-08 20:53:50,060 METRO INFO: Update config parameter num_attention_heads: 12 -> 4
2021-06-08 20:53:50,402 METRO INFO: Init model from scratch.
2021-06-08 20:53:50,452 METRO INFO: Update config parameter num_hidden_layers: 12 -> 4
2021-06-08 20:53:50,501 METRO INFO: Update config parameter hidden_size: 768 -> 128
2021-06-08 20:53:50,570 METRO INFO: Update config parameter num_attention_heads: 12 -> 4
2021-06-08 20:53:50,764 METRO INFO: Init model from scratch.
2021-06-08 20:53:57,637 METRO INFO: => loading hrnet-v2-w64 model
2021-06-08 20:53:57,694 METRO INFO: Transformers total parameters: 102256646
2021-06-08 20:53:57,756 METRO INFO: Backbone total parameters: 128059944
2021-06-08 20:53:57,867 METRO INFO: Loading state dict from checkpoint models/metro_release/metro_h36m_state_dict.bin
2021-06-08 20:54:07,573 METRO INFO: Training parameters Namespace(arch='hrnet-w64', config_name='', data_dir='datasets', device=device(type='cuda', index=0), distributed=True, drop_out=0.1, hidden_feat_dim='1024,256,128', hidden_size=128, img_scale_factor=1, input_feat_dim='2051,512,128', intermediate_size=-1, joints_loss_weight=1000.0, legacy_setting=True, local_rank=0, logging_steps=1000, lr=0.0001, model_name_or_path='metro/modeling/bert/bert-base-uncased/', num_attention_heads=4, num_gpus=8, num_hidden_layers=4, num_train_epochs=200, num_workers=24, output_dir='output/', per_gpu_eval_batch_size=30, per_gpu_train_batch_size=30, resume_checkpoint='models/metro_release/metro_h36m_state_dict.bin', run_eval_only=True, seed=88, train_yaml='imagenet2012/train.yaml', val_yaml='human3.6m/valid.protocol2.yaml', vertices_loss_weight=100.0, vloss_w_full=0.33, vloss_w_sub=0.33, vloss_w_sub2=0.33)
2021-06-08 20:56:01,317 METRO INFO: Validation epoch: 0  mPVE:   0.00, mPJPE:  54.04, PAmPJPE:  36.75 
```

**Comment by ehedlin** [comment]
I am loading the model into another project where I am using my own dataloader etc. I am assuming I am just having a problem with the disconnect.
